DependencyInjection/MisdPhoneNumberExtension.php

```diff
@@ -30,5 +31,17 @@ public function load(array $configs, ContainerBuilder $container)
     {
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('services.xml');
+
+
+        $def = $container->getDefinition('libphonenumber.phone_number_util');
+        if (method_exists($def, 'setFactory')) {
+            // to be inlined in services.xml when dependency on Symfony DependencyInjection is bumped to 2.6
+            $def->setFactory(array($container->getParameter('libphonenumber.phone_number_util.class'), 'getInstance'));
```

Comment:
I think you should not call `getParameter()` here, but use `%libphonenumber.phone_number_util.class%` instead.